src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -512,7 +512,7 @@
   <data name="TabStripDragAndDropUIOverrideCaption" xml:space="preserve">
     <value>Move tab here</value>
   </data>
-  <data name="syncStatusColumn.Header" xml:space="preserve">
+  <data name="Status" xml:space="preserve">
     <value>Status</value>
   </data>
```

Comment:
Found that this isn't used anymore
```suggestion
```

Reply:
I was considering removing it because I couldn't find anything related to it.

Reply:
I just saw that it's for the status column in the detailed layout.

Reply:
Ah sorry i missed, lets revert this change.

Reply:
Even I didn't see it.